hack/jenkins/build.sh

```diff
@@ -40,14 +40,8 @@ export CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true
 : ${FEDERATION:="false"}
 : ${KUBE_RELEASE_RUN_TESTS:="n"}
 
-# New kubernetes/release/push-ci-build.sh values
-# RELEASE_INFRA_PUSH=true when we're using kubernetes/release/push-ci-build.sh
-: ${RELEASE_INFRA_PUSH:="false"}
-# SET_NOMOCK_FLAG=true means we're doing full pushes and we pass --nomock to 
-# push-ci-build.sh. This is set to false in the
-# testing jobs and only used in the RELEASE_INFRA_PUSH=true scope below.
-: ${SET_NOMOCK_FLAG:="true"}
-export KUBE_RELEASE_RUN_TESTS RELEASE_INFRA_PUSH FEDERATION SET_NOMOCK_FLAG
+# New kubernetes/release/push-build.sh values
+export KUBE_RELEASE_RUN_TESTS FEDERATION
```

Comment:
do we need to export FEDERATION? or is it only used for the `--federation` flag below?